include/bluetooth/services/ras.h

```diff
@@ -477,6 +486,51 @@ int bt_ras_rreq_rd_overwritten_subscribe(struct bt_conn *conn, bt_ras_rreq_rd_ov
  */
 int bt_ras_rreq_rd_overwritten_unsubscribe(struct bt_conn *conn);
 
+/** @brief Subevent header callback. Called with step header for each step.
```

Comment:
Use imperative form for the function descriptions, see https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/dev_model_and_contributions/documentation/styleguide.html#functions